### PR TITLE
Goodbye LlamaKit

### PR DESCRIPTION
**Pull request by ishkawa**
I have replaced LlamaKit with Result, which is announced as an alternative of LlamaKit officially.

### CODE REVIEW

File: APIKit/API.swift [hunk 1]
```diff
@@ -83,22 +84,22 @@ public class API {
                 if !contains(self.acceptableStatusCodes, statusCode) {
                     let error: NSError = {
                         switch self.responseBodyParser.parseData(data) {
-                        case .Success(let box): return self.responseErrorFromObject(box.unbox)
-                        case .Failure(let box): return box.unbox
+                        case .Success(let box): return self.responseErrorFromObject(box.value)
+                        case .Failure(let box): return box.value
```
Comment (ikesyo) [inline APIKit/API.swift]: `Result.analysis` can be used here:

```swift
let error = self.responseBodyParser.parseData(data).analysis(ifSuccess: { value in
    return self.responseErrorFromObject(value)
}, ifFailure: { $0 })
```

File: APIKit/API.swift [hunk 2]
```diff
@@ -75,7 +76,7 @@ public class API {
             task.request = Box(request)
             task.completionHandler = { data, URLResponse, connectionError in
                 if let error = connectionError {
-                    dispatch_async(mainQueue, { handler(failure(error)) })
+                    dispatch_async(mainQueue, { handler(Result.failure(error)) })
```
Comment (ikesyo) [inline APIKit/API.swift]: These can be inferred and expressed as `.failure(error)`. Which do you prefer? @ishkawa

**Comment by ikesyo** [comment]
Other than those, looks good! :smile:

File: APIKit.xcodeproj/xcshareddata/xcschemes/APIKit-iOS.xcscheme [hunk 3]
```diff
@@ -68,7 +68,7 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       launchStyle = "0"
       useCustomWorkingDirectory = "NO"
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```
Comment (ikesyo) [inline APIKit.xcodeproj/xcshareddata/xcschemes/APIKit-iOS.xcscheme]: Is this change really needed?

Comment (ishkawa) [inline APIKit.xcodeproj/xcshareddata/xcschemes/APIKit-iOS.xcscheme]: No, this was changed by my mistake :bow:

File: APIKit/API.swift (hunk 2 above)
Comment (ishkawa) [inline APIKit/API.swift]: Cool. I prefer simpler expression to redundant one.

**Comment by ishkawa** [comment]
Thank you for reviewing! I've just fixed them and will ship this.
